src/guake/prefs.py

```diff
@@ -848,14 +1420,6 @@ def load_configs(self):
         self.set_palette_name(value)
         self.set_palette_colors(value)
 
-        # cursor shape
```

Comment:
please review your rebase, you are removing my recent changes

use git pull --rebase